internal/database/users.go

```diff
@@ -1232,6 +1234,15 @@ FROM users u %s`, query)
 }
 
 const userForSCIMQueryFmtStr = `
+WITH scim_accounts AS (
+    SELECT
+        user_id,
+        account_id,
+        account_data
+    FROM user_external_accounts
+    WHERE service_type = 'scim'
+      AND deleted_at IS NULL
+)
```

Comment:
inconsistent indentation
```suggestion
        account_data
    FROM user_external_accounts
    WHERE service_type = 'scim'
        AND deleted_at IS NULL
)
```